src/styles/aiReport.print.css

```
@@ -0,0 +1,295 @@
+  /* 표지 */
+  .ai-report-cover {
+    position: relative;
+    z-index: 3;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    min-height: 100vh;
+    padding: 14mm var(--print-margin-x) 0;
+    background: var(--color-surface-100);
+    page-break-after: always;
```

Stylelint (17.12.0)

[error] Expected "page-break-after" to be "break-after" (property-no-deprecated)

(property-no-deprecated)